### PR TITLE
Update source description for logstash collector

### LINKED ISSUE #13814: need a better description for logstashCollector's source attribute
https://draft-openlibertyio.mybluemix.net/docs/20.0.0.9/reference/config/logstashCollector.html just says:
Specifies a source to be used by the logstash collector.

Need to say what the valid values are.  more like consoleSource or messageSource descriptions from 
https://openliberty.io/docs/20.0.0.9/reference/config/logging.html

### PR DESCRIPTION
**Pull request by Yushan-Lin**
Fixes #13814

### CODE REVIEW

**Comment by Yushan-Lin** [comment]
#build

File: dev/com.ibm.ws.logstash.collector/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -6,7 +6,7 @@ logstashCollector=Logstash Collector
 logstashCollector.desc=Logstash collector gathers data from various sources and forwards the data to a logstash server using Lumberjack protocol.
 
 source=Source
-source.desc=Specifies a source to be used by the logstash collector.
+source.desc=The list of comma-separated sources used by logstash collector. Valid values are message, trace, accessLog, ffdc, garbageCollection and audit. Note: To use the audit source, enable the Liberty audit feature.
```
Comment (ManasiGandhi) [inline dev/com.ibm.ws.logstash.collector/resources/OSGI-INF/l10n/metatype.properties:9]: Change,
"The list of comma-separated sources used by logstash collector. Valid values are message, trace, accessLog, ffdc, garbageCollection and audit. Note: To use the audit source, enable the Liberty audit feature."
to,
"Specifies a list of comma-separated sources that are used by the logstash collector. Valid values are message, trace, accessLog, ffdc, garbageCollection, and audit. To use the audit source, enable the Liberty audit feature."

**Review by ManasiGandhi — COMMENTED** [review]
Some edit suggestions.

File: dev/com.ibm.ws.logstash.collector/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (Yushan-Lin) [reply dev/com.ibm.ws.logstash.collector/resources/OSGI-INF/l10n/metatype.properties]: Just updated the description

**Comment by Yushan-Lin** [comment]
#build

File: dev/com.ibm.ws.logstash.collector/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -6,7 +6,7 @@ logstashCollector=Logstash Collector
 logstashCollector.desc=Logstash collector gathers data from various sources and forwards the data to a logstash server using Lumberjack protocol.
 
 source=Source
-source.desc=Specifies a source to be used by the logstash collector.
+source.desc=Specifies a list of comma-separated sources that are used by the logstash collector. Valid values are message, trace, accessLog, ffdc, garbageCollection, and audit. To use the audit source, enable the Liberty audit feature.
```
Comment (donbourne) [inline dev/com.ibm.ws.logstash.collector/resources/OSGI-INF/l10n/metatype.properties:9]: it's odd that we say to enable the audit feature to get audit events, but we don't mention that you need an accessLogging element if you want accessLog events.  I'd suggest adding a sentence to cover that (and also adding that to the descriptions for the messageSource / consoleSource of logging)

**Comment by Yushan-Lin** [comment]
#build

**Review by donbourne — APPROVED** [review]

**Comment by Yushan-Lin** [comment]
#run-libby-bot

**Pull request merged**